apps/costcontrol/js/settings/settings.js

```diff
@@ -128,8 +135,12 @@ var Settings = (function() {
         closeSettings();
       });
 
-      function _setResetTimeToDefault(value, old, key, settings) {
-        var today = new Date();
+      function _setDataPeriodBounds(value, old, key, settings) {
+        var today = InputParser.exportDate(new Date());
+        if (settings.startingTime !== today) {
+          ConfigManager.setOption({ startingTime: today });
+        }
+        today = new Date();
```

Comment:
Please, differentiate variables for `today = new Date()` and the exported value such as `exportedDate = InputParser.exportDate(today);`.